### PR TITLE
Respond to ruby study prompts

### PR DESCRIPTION
**Pull request by cuprous**

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+007.to_s.ljust(3, "0")
```
Comment (RealWeeks) [inline study.md:22]: .to_s.reverse!

**Comment by RealWeeks** [comment]
Good job

File: study.md (hunk 1 above)
Comment (cuprous) [inline study.md:22]: 007.to_s.reverse! results in an output of "7" and not "700" since irb and pry strip the leading zeros of the integer.

**Comment by RealWeeks** [comment]
Thanks, will fix going forward.

**Pull request closed without merging**